src/trimmer.js

```diff
@@ -27,7 +27,9 @@ class TextTrim extends React.Component {
 
 		window.addEventListener( 'resize', this.resize );
 
-		this.resize();
+		window.setTimeout(()=>{
+            this.resize();   
+        },10)
```

Comment:
Is 0 enough here to bump it to the end of stack?

Reply:
yes, 0 is enough. I'll pull again.